### ISSUE TITLE
[Bug]: stdlib logo on landing page has insufficient contrast when viewed in dark mode

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#51]
When viewing the landing page in dark mode, the logo tag line has insufficient contrast with the background. We should probably target the text using, e.g., the default theme text color.

![Screen Shot 2022-05-20 at 5 08 16 PM](https://user-images.githubusercontent.com/2643044/169626306-ee039c58-a1cd-4823-a7ff-fabfc3ef7688.png)

**Comment by kgryte** [comment]
The issue for resolving this is that the banner is loaded via an `img` tag: https://github.com/stdlib-js/www/blob/8107e2fa8925c7c96608cc15500af5fac55505e9/src/components/welcome/banner.jsx#L36

That's problematic for theming as we cannot modify SVG attributes for SVG loaded this way.

The rationale for loading via an `img` tag was to reduce bundle sizes and leverage caching.

Likely best solution here is to use the existing `logo.jsx` component and then add the tag line as a separate DOM element which we can then theme.

**Comment by kgryte** [comment]
![Screen Shot 2022-05-21 at 4 22 56 AM](https://user-images.githubusercontent.com/2643044/169649559-77c839ac-9a25-4ed6-938a-bee69a1aec7d.png)
![Screen Shot 2022-05-21 at 4 23 08 AM](https://user-images.githubusercontent.com/2643044/169649560-ba9c0194-d0e5-47b0-b5a9-e5d5f0ec63df.png)
